flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/rules/FlinkStreamRuleSets.scala

```diff
@@ -486,7 +486,13 @@ object FlinkStreamRuleSets {
     // incremental agg rule
     IncrementalAggregateRule.INSTANCE,
     // optimize window agg rule
-    TwoStageOptimizedWindowAggregateRule.INSTANCE
-  )
+    TwoStageOptimizedWindowAggregateRule.INSTANCE,
+    // simplify window tvf
+    SimplifyWindowTableFunctionRule.WITH_CALC_WINDOW_RANK,
+    SimplifyWindowTableFunctionRule.WITH_WINDOW_RANK,
+    SimplifyWindowTableFunctionRule.WITH_LEFT_RIGHT_CALC_WINDOW_JOIN,
+    SimplifyWindowTableFunctionRule.WITH_LEFT_CALC_WINDOW_JOIN,
+    SimplifyWindowTableFunctionRule.WITH_RIGHT_CALC_WINDOW_JOIN,
+    SimplifyWindowTableFunctionRule.WITH_WINDOW_JOIN)
```

Comment:
I didn't put into `PHYSICAL_OPT_RULES ` because the rules could not work well in Cost-based-optimization phase.
Because equivalent node didn't have less cost.